catroid/src/main/java/org/catrobat/catroid/ui/recyclerview/dialog/ReplaceExistingProjectDialogFragment.java

```diff
@@ -122,20 +117,30 @@ public void onPositiveButtonClick(DialogInterface dialog, String textInput) {
 				.setNegativeButton(R.string.notification_download_project_cancel, null)
 				.create();
 
-		radioGroup.setOnCheckedChangeListener(new RadioGroup.OnCheckedChangeListener() {
-			@Override
-			public void onCheckedChanged(RadioGroup group, @IdRes int checkedId) {
-				switch (checkedId) {
-					case R.id.replace:
-						inputLayout.setVisibility(TextView.GONE);
-						alertDialog.getButton(AlertDialog.BUTTON_POSITIVE).setEnabled(true);
-						break;
-					case R.id.rename:
-						inputLayout.setVisibility(TextView.VISIBLE);
-						alertDialog.getButton(AlertDialog.BUTTON_POSITIVE)
-								.setEnabled(textWatcher.validateInput(inputLayout.getEditText().toString(), getContext()) == null);
-						break;
-				}
+		radioGroup.setOnCheckedChangeListener((group, checkedId) -> {
+			switch (checkedId) {
+				case R.id.replace:
+					inputLayout.setVisibility(TextView.GONE);
+					alertDialog.getButton(AlertDialog.BUTTON_POSITIVE).setEnabled(true);
+					break;
+				case R.id.rename:
+					inputLayout.setVisibility(TextView.VISIBLE);
+					alertDialog.getButton(AlertDialog.BUTTON_POSITIVE)
+							.setEnabled(textWatcher.validateInput(inputLayout.getEditText().getText().toString(), getContext()) == null);
+					break;
+			}
+		});
+
+		alertDialog.setOnShowListener(dialog -> {
```

Comment:
Please remove this method as it is just duplicate code (have a look at the `TextInputDialog.Builder` class where the same functionality is already implemented.